### ISSUE TITLE
High priority?

### ISSUE THREAD

**Issue by RobStallion** [dwyl/phoenix-ecto-append-only-log-example#22]
@nelsonic based on the following comment you made in [here](https://github.com/dwyl/time/issues/210)...

> The work to make Phoenix append-only is "on-going". Once CID is complete we will need to integrate it into our Example: https://github.com/dwyl/phoenix-ecto-append-only-log-example and then add it to Alog (which will need to be re-worked to make the log "generic" ... tbd.)

Would I be correct in assuming that now that CID has been published, updating this example is the next priority (one of the next priorities) on the list?

**Comment by nelsonic** [comment]
@RobStallion thanks for opening this issue. I was _about_ to create it ... 👍 
Please estimate and proceed. thanks. ✨

**Comment by RobStallion** [comment]
@nelsonic I have had a brief read through the current example and have a few question/thoughts on how things might/will change when we use CIDs. 

I am going to open separate issues for each function so that it is easier for people to comment with their thoughts for how that particular function will work.

**Comment by RobStallion** [comment]
@nelsonic Yeah I'd be up for that. I'll write up my thoughts in issues first though so that others can comment and maybe join if that makes sense 👍
